src/obfl-specification.html

```diff
@@ -1266,6 +1268,47 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd>The total height.</dd>
       </dl>
     </dd>
+    <dd>
+      <p>Some of the above-mentioned variables can be used in use-when
+        conditions. Here is an overview of all elements with a
+        use-when condition and the variables that can be used in the
+        condition:</p>
```

Comment:
I really don't understand why you added this here. Don't get me wrong, it's a nice little table, but why put it here in this section? The reader is not interested in this when he is reading about the `evaluate` element. Same goes for the variables for the `cost` attribute.